audit_vms.py

```diff
@@ -73,6 +75,7 @@ def parse_args(argv=None):
                                      description=__doc__, allow_abbrev=False)
     parser.add_argument('--list-running-for-hours', type=float, help='List VMs running for at least X hours')
     parser.add_argument('--delete-ghost-vms', action='store_true', help='Require --list-running-for-hours')
+    parser.add_argument('--force-delete', default=False, action='store_true', help='For bypass interactive confirmation')
```

Comment:
👍 
Le "For" n'est pas nécessaire en anglais je crois ici

Reply:
👍 Yes